web/themes/custom/badcamp2017/css/styles.css

```diff
@@ -5312,12 +5324,16 @@ a.site-slogan {
   }
 }
 
+#block-builtby p {
```

Comment:
similar issues as previous comment-- using an ID. 

For more info on class naming conventions, please see http://johnalbin.github.io/flower-power/section-flower.html